### ISSUE TITLE
rationalize content models of org and place and person

### ISSUE THREAD

**Issue by TEITechnicalCouncil** [TEIC/TEI#367]
As discussed in ticket http://purl.org/TEI/FR/3440977, the content models of &lt;place&gt; and &lt;org&gt; are unnecessarily messy, and would benefit from being rationalized by the creation of a model.placePart and model.orgPart respectively (along the lines of model.personPart).

It might even be the case that some subset of the overlap of these three models could be defined as a smaller model (entityPart?) that all such elements would inherit (containing, e.g., idno), so that any new entityLike element in future can inherit them as a quick start.

While we're at it, we should look at whether we can also create *Part models for the elements &lt;object&gt;, &lt;event&gt; and &lt;nym&gt;, all of which arguably want to inherit &lt;idno&gt; for the same reason as person and place; all can have names, bibliography, relations, etc.

Original comment by: @gabrielbodard

**Comment by emylonas** [comment]
This should be considered with #562

**Comment by raffazizzi** [comment]
Council met with Christian-Emil Ore to coordinate work on this very topic.
Council subgroup agrees that common parts of -ographies can be grouped into a new model, but we think this is best handled as part of larger reform of modelling of ontologies in TEI and Christian-Emil Ore will provide a report on that. Subgroup suggests moving to Status: Blocked again.

**Comment by raffazizzi** [comment]
F2F Marked status Go again. @sydb to figure out and propose a rationalization.

**Comment by martindholmes** [comment]
@sydb I suggest we devote one of our Friday meetings to kick-starting this after the next release is out.

**Comment by martindholmes** [comment]
@sydb We seem to have dropped the ball on this. Shall we address it this Friday?

**Comment by martindholmes** [comment]
@sydb Ping on this dormant ticket! If we're not going ahead with it, should we just close it?

**Comment by jamescummings** [comment]
I don't think we should close this ticket but instead revitalise it.  Ensuring as much as possible consistency of potential encoding across person/place/org and I would also add event, is a desirable goal. This has been done piecemeal with issues like https://github.com/TEIC/TEI/issues/1756 but we should step back, re-assess what the differences are between them and see how they could be better rationalised.

**Comment by martindholmes** [comment]
@jamescummings Coming back to this, I feel that there's not enough of a clear action plan for this to be a ticket. It isn't responding to any specific problems that we can get our teeth into. It might be appropriate as a charge for a working group, but as a ticket, I don't know where to go with it.

**Comment by martindholmes** [comment]
@jamescummings Re-reading again, I find, astonishingly enough, that I continue to agree with myself: this ticket is too vast and sprawling to give birth to a useful action plan. Meanwhile, existing projects are so thoroughly invested in the current structures of the most common entity-like elements that we would have a hard time implementing any significant change. If anything, this is perhaps a structural goal for those who end up working on P6, where rational re-imagining will be practical and (I hope) encouraged. Let's close this one.

**Comment by ebeshero** [comment]
We are closing this in favor of reconsidering the "-ography" elements' content model for P6.
